packages/aws-fargate/test/esm/test.js

```diff
@@ -0,0 +1,197 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const { expect, assert } = require('chai');
+const { fail } = assert;
+const path = require('path');
+const constants = require('@instana/core').tracing.constants;
+
+const Control = require('../Control');
+const { expectExactlyOneMatching } = require('../../../core/test/test_util');
+const config = require('../../../serverless/test/config');
+const retry = require('../../../serverless/test/util/retry');
+
+const downstreamDummyPort = 4567;
+const downstreamDummyUrl = `http://localhost:${downstreamDummyPort}/`;
+const fs = require('fs');
+const region = 'us-east-2';
+const account = '555123456789';
+const instrumentedContainerName = 'nodejs-fargate-test-container';
+const taskArn = `arn:aws:ecs:${region}:${account}:task/55566677-c1e5-5780-9806-aabbccddeeff`;
+const instrumentedContainerId = `${taskArn}::${instrumentedContainerName}`;
+
+const containerAppPath = path.join(__dirname, './app.mjs');
+const instanaAgentKey = 'aws-fargate-dummy-key';
+
+const requestHeaders = {
+  'X-Entry-Request-Header-1': 'entry request header value 1',
+  'X-Entry-Request-Header-2': ['entry', 'request', 'header', 'value 2'],
+  'X-Entry-Request-Header-3': 'not configured to capture this',
+  'X-Entry-Request-Header-4': ['not', 'configured', 'to', 'be', 'captured']
+};
+
+function prelude(opts = {}) {
+  this.timeout(config.getTestTimeout());
+  this.slow(config.getTestTimeout() / 2);
+  if (__dirname) {
+    const files = fs.readdirSync(__dirname);
+    const esmApp = files.find(f => f.indexOf('.mjs') !== -1);
+    if (esmApp) {
+      opts.execArgv = [`--experimental-loader=${path.join(__dirname, '..', '..', 'esm-loader.mjs')}`];
```

Comment:
TBD: How does that work? The `Control` class does not check for `execArgv`. It is hardcoded to `--require`.

Reply:
changed the code, these modifications were added for testing purposes but should have been removed